modules/RTC/BridgeChannel.js

```diff
@@ -344,11 +344,11 @@ export default class BridgeChannel {
 
                 break;
             }
-            case 'SelectedUpdateEvent': {
-                const isSelected = obj.isSelected;
+            case 'SenderVideoConstraintsChangedEvent': {
+                const idealHeight = obj.videoConstraints.idealHeight;
```

Comment:
is it possible for Websocket data channel to be established and receive this event, before JVB media session is established?